partiql-eval/src/plan.rs

```diff
@@ -0,0 +1,110 @@
+use crate::eval;
+use crate::eval::{
+    EvalBinop, EvalBinopExpr, EvalExpr, EvalLitExpr, EvalPath, EvalVarRef, Evaluable, TupleSink,
+};
+use partiql_logical as logical;
+use partiql_logical::{BinaryOp, BindingsExpr, PathComponent, ValueExpr};
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+pub struct EvaluatorPlanner {
+    pub output: Rc<RefCell<dyn TupleSink>>,
+}
+
+impl EvaluatorPlanner {
+    pub fn compile(&self, be: BindingsExpr) -> Box<dyn Evaluable> {
+        self.plan_eval(be)
+    }
+
+    #[inline]
+    fn plan_eval(&self, be: BindingsExpr) -> Box<dyn Evaluable> {
+        match be {
+            BindingsExpr::From(logical::From {
+                expr,
+                as_key,
+                at_key: _,
+                out,
+            }) => Box::new(eval::EvalFrom::new(
+                self.plan_values(expr),
+                &as_key,
+                self.plan_bindings(*out),
+            )),
+            _ => panic!("Unevaluable bexpr"),
```

Comment:
To check my understanding, only `BindingsExpr::From` is currently supported/implemented in the POC's evaluator planner?